tracer/src/Datadog.InstrumentedAssemblyGenerator/MetadataNameParser.cs

```diff
@@ -18,6 +19,8 @@ internal class MetadataNameParser
 {
     internal static MetadataMember Parse(Token token, string name)
     {
+        // TODO: Add parent type (declaring type) to be able to identify two nested types with the same name -- see ProfilerMetadataImporter.ImportInstrumentedAssemblyMetadata
+        // https://datadoghq.atlassian.net/browse/DEBUG-1029
```

Comment:
Please remove the internal JIRA link - we should not include these links in an open source repo.

Reply:
Fixed in 871a652d9b150d63443d87a542e34d4643145adf